src/components/Header/Header.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+
+import styles from './Header.module.scss'
+import logo from '../../img/logo.png'
+
+import NavBar from '../Navbar/Navbar'
```

Comment:
Please use correct import order. You can find correct import order in `Post` component.
Example: 
```
venors imports
[empty line]
components/containers imports
[empty line]
styles import
```